docs/source/yelpsoa_configs.rst

```diff
@@ -623,6 +623,11 @@ If a Tron **action** of a job is of executor type ``spark``, it MAY specify the
     https://github.com/Yelp/service_configuration_lib/blob/master/service_configuration_lib/spark_config.py#L9
     for a complete list of such configurations.
 
+  * ``aws_credentials_yaml``: Path to the yaml file containing credentials to be set in the task's
+    AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY environment variables. If the file path does not
```

Comment:
Same question as qui: Can there be a sane default here so that users don't have to continuously specify this on every action? (convention over configuration)

Reply:
The default path is handled by `get_aws_credentials` helper function. Lemme document that here, too.